index.html

```diff
@@ -25,128 +26,6 @@
     <script src="clocks/clock-12.js" type="text/javascript"></script>
 
     <script src="sketch.js" type="text/javascript"></script>
-
-    <style>
-    body {
-      padding: 0;
-      margin: 0;
-      background: #000;
-      color: #FFF;
-      font-family: arial;
-    }
-    canvas {
-      vertical-align: top;
-      width: 20vw !important;
-      height: 20vh !important;
-      width: calc(20vw - 2px) !important;
-      height: calc(20vh - 2px) !important;
-      min-width: 150px;
-      min-height: 150px;
-    }
-    a {
-      color: #fff !important;
-    }
-    #container {
-      display: grid;
-      grid-template-rows: repeat(4, 1fr);
-      grid-template-columns: repeat(4, 1fr);
-      width: 80vw;
-      min-height: 80vh;
-      margin: 10vh 10vw;
-    }
-
-    #container .clock {
-      border: 1px solid #333;
-    }
-
-    #container .clock:hover {
-      border-color: red;
-    }
-
-    #container .copyright {
-      grid-column-start: 1;
-      grid-column-end: 5;
-      padding: 0 20px;
-      text-align: right;
-    }
-
-    #container.single .clock {
-      display: none;
-    }
-
-    #container.single .clock.selected {
-      display: block;
-      grid-column-start: 1;
-      grid-column-end: 5;
-      grid-row-start: 1;
-      grid-row-end: 4;
-    }
-
-    #container.single .clock.selected canvas {
-      width: 80vw !important;
-      height: 60vh !important;
-      width: calc(80vw - 2px) !important;
-      height: calc(60vh - 2px) !important;
-    }
-
-    #container.single .copyright {
-      grid-column-end: 3;
-    }
-
-    #container .contributors {
-      display: none;
-      padding: 0 20px;
-      grid-column-start: 3;
-      grid-column-end: 5;
-    }
-
-    #container.single .contributors {
-      display: block;
-    }
-
-    ul {
-      margin: 0;
-      padding: 0;
-    }
-
-    li {
-      display: inline;
-
-    }
-
-    li + li::before {
-      content: ", ";
-    }
-
-    @media (max-width: 800px) {
-      #container {
-        grid-template-rows: repeat(7, 1fr);
-        grid-template-columns: repeat(2, 1fr);
-      }
-      #container.single {
-        grid-template-rows: 1fr 3fr 1fr;
-        grid-template-columns: repeat(2, 1fr);
-      }
-      #container .copyright {
-        grid-row-start: 1;
-        grid-column-end: 3;
-      }
-      canvas {
-        width: 40vw !important;
-      }
-      #container.single .clock.selected {
-        grid-column-end: 3;
-        grid-row-start: 2;
-        grid-row-end: 3;
-      }
-      #container .contributors {
-        grid-row-start: 3;
-        grid-column-start: 1;
-        grid-column-end: 3;
-      }
-    }
-
-    </style>
```

Comment:
We need to link to the styles externally with a link element fro this to work

Reply:
forgot to do that 😆